src/System.Private.CoreLib/shared/System/Collections/Generic/Dictionary.cs

```diff
@@ -831,12 +839,13 @@ public bool Remove(TKey key, out TValue value)
                 ThrowHelper.ThrowArgumentNullException(ExceptionArgument.key);
```

Comment:
We should add TODO-NULLABLE-GENERIC to the `public bool Remove(TKey key, out TValue value)` signature, as `TValue` could be null if `Remove` returns false.  Maybe https://github.com/dotnet/roslyn/issues/26761 as well.